invenio_assets/cli.py

```diff
@@ -86,6 +88,11 @@ def npm(package_json, output_file):
                         click.echo)
     output['dependencies'].update(deps)
 
+    # Load pinned dependencies
+    if pinned_file:
+        pinned_deps = dict(output, **json.load(pinned_file))
```

Comment:
Do you need this intermediate copy? Can we just update the `output['dependencies'].update(json.load(pinned_file).get('dependencies', {}))`?